elementor/elementor
# Fix: Autoplay for video's [ED-14512]

## PR Checklist



## PR Type
What kind of change does this PR introduce?


## Summary

This PR can be summarized in the following changelog entry:

* Added the support to autoplay video's for normal video's and video's with a overlay image (lightbox included).

## Description
An explanation of what is done in this PR

* This fixes the autoplay for chrome browsers. And added a note for the chrome autoplay policy. (New translatable string)

## Test instructions
This PR can be tested by following these steps:

* 1. Create a video element.
* 2. Enable autoplay
* 3. See that autoplay works
* 4. Enable overlay image
* 5. See that autoplay works
* 6. Enable lightbox
* 7. See that autoplay works

## Quality assurance


Fixes #

#24273, #14437, #17080, #7964
